src/web_app/js/peerconnectionclient.js

```diff
@@ -43,6 +43,14 @@ var PeerConnectionClient = function(params, startTime) {
   this.pc_.onsignalingstatechange = this.onSignalingStateChanged_.bind(this);
   this.pc_.oniceconnectionstatechange =
       this.onIceConnectionStateChanged_.bind(this);
+  window.dispatchEvent(new CustomEvent('pccreated', {
+    detail: {
+      pc: this,
```

Comment:
Yeah, I want to avoid calling the underlying `pc_` object directly, I'd rather call the `getPeerConnectionStats` function on `this` because since other parts of the app uses it I believe it will be better maintained.